pkg/MetaMorphoAction.ts

```diff
@@ -2,119 +2,219 @@ import { BigNumberish } from "ethers";
+   * @param fee The new fee percentage (in WAD).
+   */
+  export function setFee(fee: BigNumberish): MetaMorphoCall {
+    return METAMORPHO_IFC.encodeFunctionData("setFee", [fee]);
   }
 
   /* TIMELOCK */
 
-  static submitTimelock(newTimelock: BigNumberish): MetaMorphoCall {
-    return MetaMorphoAction.METAMORPHO_IFC.encodeFunctionData("submitTimelock", [newTimelock]);
+  /**
+   * Encodes a call to a MetaMorpho instance to submit a new timelock.
+   * @param newTimelock The new timelock (in seconds).
+   */
+  export function submitTimelock(newTimelock: BigNumberish): MetaMorphoCall {
+    return METAMORPHO_IFC.encodeFunctionData("submitTimelock", [newTimelock]);
   }
 
-  static acceptTimelock(): MetaMorphoCall {
-    return MetaMorphoAction.METAMORPHO_IFC.encodeFunctionData("acceptTimelock");
+  /**
+   * Encodes a call to a MetaMorpho instance to accept the pending timelock.
+   */
+  export function acceptTimelock(): MetaMorphoCall {
+    return METAMORPHO_IFC.encodeFunctionData("acceptTimelock");
   }
 
-  static revokePendingTimelock(): MetaMorphoCall {
-    return MetaMorphoAction.METAMORPHO_IFC.encodeFunctionData("revokePendingTimelock");
+  /**
+   * Encodes a call to a MetaMorpho instance to revoke the pending timelock.
+   */
+  export function revokePendingTimelock(): MetaMorphoCall {
+    return METAMORPHO_IFC.encodeFunctionData("revokePendingTimelock");
   }
 
   /* SUPPLY CAP */
 
-  static submitCap(marketParams: MarketParamsStruct, newSupplyCap: BigNumberish): MetaMorphoCall {
-    return MetaMorphoAction.METAMORPHO_IFC.encodeFunctionData("submitCap", [marketParams, newSupplyCap]);
+  /**
+   * Encodes a call to a MetaMorpho instance to submit a new supply cap.
+   * @param marketParams The market params of the market of which to submit a supply cap.
+   * @param newSupplyCap The new supply cap.
+   */
+  export function submitCap(marketParams: MarketParamsStruct, newSupplyCap: BigNumberish): MetaMorphoCall {
+    return METAMORPHO_IFC.encodeFunctionData("submitCap", [marketParams, newSupplyCap]);
   }
 
-  static acceptCap(marketParams: MarketParamsStruct): MetaMorphoCall {
-    return MetaMorphoAction.METAMORPHO_IFC.encodeFunctionData("acceptCap", [marketParams]);
+  /**
+   * Encodes a call to a MetaMorpho instance to accept the pending supply cap.
+   * @param marketParams The market params of the market of which to accept the pending supply cap.
+   */
+  export function acceptCap(marketParams: MarketParamsStruct): MetaMorphoCall {
+    return METAMORPHO_IFC.encodeFunctionData("acceptCap", [marketParams]);
   }
 
-  static revokePendingCap(id: string): MetaMorphoCall {
-    return MetaMorphoAction.METAMORPHO_IFC.encodeFunctionData("revokePendingCap", [id]);
+  /**
+   * Encodes a call to a MetaMorpho instance to revoke the pending supply cap.
+   * @param id The id of the market of which to revoke the pending supply cap.
+   */
+  export function revokePendingCap(id: string): MetaMorphoCall {
```

Comment:
```suggestion
  export function revokePendingCap(id: MarketId): MetaMorphoCall {
```
from `bundler-sdk`, but Idk if we want to have this dependency

Reply:
I don't think so for now at least